### ISSUE TITLE
Accuracy of `model_arch==3` i.e. `own_model`

### ISSUE THREAD

**Issue by deeiip** [robmsmt/KerasDeepSpeech#3]
1. Is there any result on any dataset for your own model i.e. `model_arch == 3`?
2. Secondly, If I select model_acrh == 3. The console prints it as DS3. I dont suppose it this [model](https://arxiv.org/abs/1707.07413), 
Thanks in advance.

**Comment by robmsmt** [comment]
Currently (over the next week or so) we are rewriting the project from scratch so it's more stable and has better _modular model_ design. This means that each model file will live in the models folder which contains the layers, structure and any model settings. You can then download each with a summary of what it was trained on, (language, dataset etc), and resulting WER.

With regards to the 1st point, we intend to release all results (along with each model file) for all models because that's one of the areas that tends to annoy me when researching ASR. Specifically, you never know what doesn't work well, you only get a very limited view of "here's a model that works well on this data". 

For the 2nd, it is not that model, it's something I created. Thanks for the reminder, it should have a different name ideally, not DS3.

R

**Comment by dipanjannag** [comment]
If you can provide all the models with specifications that would be great.
I feel the issue is also because, most projects available, is mostly using more bare metal math library like tensorflow. And it makes it hard to experiment with.
I'll be waiting for the update. And if you have any [help wanted] item, please let me know. I'd be happy to help
